tests/foreman/ui/test_sync_templates.py

```diff
@@ -0,0 +1,125 @@
+"""Test class for Foreman Templates Import Export UI
+
+:Requirement: TemplatesPlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Integration
+
+:CaseComponent: TemplatesPlugin
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo import ssh
+from robottelo.constants import FOREMAN_TEMPLATES_COMMUNITY_URL
+from robottelo.decorators import tier2
+from robottelo.decorators import upgrade
+
+
+@pytest.fixture(scope='module')
+def templates_org():
+    return entities.Organization().create()
+
+
+@pytest.fixture(scope='module')
+def templates_loc(templates_org):
+    return entities.Location(organization=[templates_org]).create()
+
+
+@tier2
+@upgrade
+def test_positive_import_templates(session, templates_org, templates_loc):
+    """Import template(s) from external source to satellite
+
+    :id: 524bf384-703f-48a5-95ff-7c1cf97db694
+
+    :Steps:
+
+        1. Navigate to Host -> Sync Templates, and choose Import.
+        2. Select fields:
+            associate = always
+            filter = Alterator default PXELinux
+            prefix = <any_prefix>
+            repo = Community Repo
+        3. Submit to Import the template from community repo.
+
+    :expectedresults:
+
+        1. The reports are displayed for templates imported / not imported.
+        2. The filter provisioning template is imported and assigned to current taxonomies
+
+    :CaseImportance: Critical
+    """
+    import_template = 'Alterator default PXELinux'
+    prefix_name = gen_string('alpha', 8)
+    with session:
+        session.organization.select(org_name=templates_org.name)
+        session.location.select(loc_name=templates_loc.name)
+        import_title = session.sync_template.sync(
+            {
+                'sync_type': 'Import',
+                'template.associate': 'Always',
+                'template.filter': import_template,
+                'template.prefix': f'{prefix_name} ',
```

Comment:
`f'{prefix_name} '` was that extra space intentional

Reply:
Yes ! intentional.